deploy/criReturnStepFunction.asl.json

```diff
@@ -89,10 +89,23 @@
           "Variable": "$.lambdaResult.journey",
           "StringEquals": "/journey/ci-scoring",
           "Next": "CiScoring"
+        }
+      ],
+      "Default": "ReturnToFrontend"
+    },
+    "ScoringResponse": {
+      "Type": "Choice",
+      "Choices": [
+        {
+          "Not": {
+            "Variable": "$.lambdaResult.journey",
+            "IsPresent": true
+          },
+          "Next": "ReturnToFrontend"
         },
         {
           "Variable": "$.lambdaResult.journey",
-          "StringEquals": "/journey/evaluate",
+          "StringEquals": "/journey/next",
```

Comment:
Just wondering if we can do without this new choice task and keep the template a bit dryer. We're not using a `next` string in the `RouteJourneyResponse` task so could use it in there?